docs/targets/awskinesis.md

```diff
@@ -3,13 +3,13 @@
 This event Target receives [CloudEvents][ce] over HTTP and publishes the event to
 AWS Kinesis.
 
-## Prerequisites
+## Prerequisite(s)
 
-1. AWS API key and secret
+- AWS API key and secret
```

Comment:
Sorry I just learned this but we can use `*` here instead of `-`. Do you mind changing this? Do you also mind changing the template and the example?

Reply:
To be fair, the only reason why I asked not to use `-` in the previous PR is that we were using `*` everywhere else, and I didn't want the PR to drift into that discussion and become larger than it should be.

With that being said, I do agree that `-` would be the more logical choice, since it eliminates the ambiguity between `*` (bulltet point), `* *` (italic) and `** **` (bold) <-- _I blame markdown for this mess_.

I think we should replace `*` with `-` in all our lists, but we should do it in one pass, as a separate PR.

WDYT?

Reply:
I think I agree with Antoine in that using `-` may be easier to differentiate from italic and bold notation. I can make the changes unless there are any strong feelings otherwise.

But I will make the numerical changes and be sure to standardize the notation for unordered lists. :)

Reply:
@antoineco I like using `-` better `*` 

@thomasgilmore95 is making a rules guideline. Thomas, do you mind adding that all unordered list should use `-` instead?

Reply:
@antoineco @daceynolan I will make the unordered list to `-` instead.

Reply:
@thomasgilmore95 I actually went ahead and changed it already in this PR if that's alright. It'll show up in my next commit.

Reply:
@alicenstar sounds good.

Reply:
@thomasgilmore95 I think @alicenstar only changed the targets so it is okay if you work on the sources you worked on yesterday :)